src/main/kotlin/io/quarkus/code/CodeQuarkusResource.kt

```diff
@@ -139,11 +139,12 @@ class CodeQuarkusResource {
             extensions: Set<String>,
 
             @DefaultValue("MAVEN")
+            @NotEmpty
             @QueryParam("b")
-            @Parameter(name = "b", description = "The build tool to use (MAVEN or GRADLE) (default: MAVEN)")
-            buildTool: io.quarkus.code.model.BuildTool
```

Comment:
You can remove the `io.quarkus.code.model.BuildTool` class now

Reply:
Good catch. I hesitated for a while before not using it, but I found it more confusing since there is one declared in Quarkus already.

The second reason is stupid, I will not say it 😅